CollapseLauncher/Classes/GameManagement/GameSettings/Honkai/RegistryClass/Enums/Enums.cs

```diff
@@ -1,25 +1,57 @@
 ﻿using System.Text.Json.Serialization;
 
+// ReSharper disable UnusedMember.Global
+// ReSharper disable InconsistentNaming
 namespace CollapseLauncher.GameSettings.Honkai.Enums
 {
     /// <summary>
     /// This selection has 4 name types: Low (0), Middle (1), High (2), VHigh (3)<br/>
     /// </summary>
     [JsonConverter(typeof(JsonStringEnumConverter<SelectResolutionQuality>))]
-    internal enum SelectResolutionQuality : int { Low, Middle, High, VHigh }
+    internal enum SelectResolutionQuality { Low, Middle, High, VHigh }
 
     /// <summary>
     /// This selection has 5 name types: DISABLED (0), LOW (1), MIDDLE (2), HIGH (3), ULTRA (4)<br/>
     /// </summary>
     [JsonConverter(typeof(JsonStringEnumConverter<SelectShadowLevel>))]
-    internal enum SelectShadowLevel : int { DISABLED, LOW, MIDDLE, HIGH, ULTRA }
+    internal enum SelectShadowLevel { DISABLED, LOW, MIDDLE, HIGH, ULTRA }
 
     /// <summary>
-    /// This selection has 3 name types: DISABLED (0), LOW (1), HIGH (2)<br/>
+    /// This selection has 4 name types: DISABLED (0), LOW (1), MIDDLE (2), HIGH (3)<br/>
     /// </summary>
     [JsonConverter(typeof(JsonStringEnumConverter<SelectReflectionQuality>))]
-    internal enum SelectReflectionQuality : int { DISABLED, LOW, HIGH }
+    internal enum SelectReflectionQuality { DISABLED, LOW, MIDDLE, HIGH }
 
+    /// <summary>
+    /// This selection has 5 name types: Off (0), Low (1), Middle (2), High (3), Ultra (4)<br/>
+    /// </summary>
+    [JsonConverter(typeof(JsonStringEnumConverter<SelectLightningQuality>))]
+    internal enum SelectLightningQuality : int { Off, Low, Middle, High, Ultra }
+    
+    /// <summary>
+    /// This selection has 4 name types: Off, Low, Middle, High
+    /// </summary>
+    [JsonConverter(typeof(JsonStringEnumConverter<SelectPostFXQuality>))]
+    internal enum SelectPostFXQuality : int { Off, Low, Middle, High }
+    
+    /// <summary>
+    /// This selection has 3 name types: Off, FXAA, TAA
+    /// </summary>
+    [JsonConverter(typeof(JsonStringEnumConverter<SelectAAType>))]
+    internal enum SelectAAType : int { Off, FXAA, TAA }
```

Comment:
I believe this logic can be applied to all of the parameters, since there are other ones such as `SelectGlobalIllumination` and `SelectAmbientOcclusion` that still have the constant type `int`.

Reply:
Fixed